website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,473 @@
+---
+title: Customize schema and alias
+id: customize-schema-alias
+description: "How to properly adjust your generate_schema_name() and generate_alias_name() macros.."
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+When we develop dbt models and run or build them, objects get created in the data warehouse.
+
+In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog.
+
+The default behavior of what object is created is the following:
+
+- The database where the object is created will be the database configured at the [environment level in dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud-environments), or in the `profiles.yml` in dbt Core.
+
+- The schema depends on whether a [custom schema](https://docs.getdbt.com/docs/build/custom-schemas) has been defined for the model:
+    - If no custom schema is defined, the schema where the object is created will be the schema that has been configured in dbt Cloud (`dbt_username` for Dev and the configured default schema for any deployment environment); or the schema in the `profiles.yml` in dbt Core.
+        - Note: Automated CI jobs are a specific case, where the schema name is derived from the job number and PR number: `dbt_cloud_pr_<job_id>_<pr_id>`
+    - If a custom schema has been defined, it will concatenate the schema from above with the custom one.
+        - For example, if the configured schema is `dbt_myschema` and the custom one is `marketing`, the objects will be created under `dbt_myschema_marketing`.
+
+- The object name depends on whether an [alias](https://docs.getdbt.com/reference/resource-configs/alias) has been defined on the model:
```

Comment:
updating link to relative paths. i suggest you do the same for the rest of the links
```suggestion
- The object name depends on whether an [alias]/reference/resource-configs/alias) has been defined on the model:
```